src/dex-ui/pages/dao/DAODashboard.tsx

```diff
@@ -93,40 +94,57 @@ export function DAODashboard(props: DAODashboardProps) {
 
   if (dao && isDAOFound && isSuccess) {
     const { accountId, type, name, logoUrl } = dao;
+    const isVotingPowerVisible = type === DAOType.GovernanceToken || type === DAOType.NFT;
```

Comment:
Note for the future: We're looking to remove the concept of using fungible tokens in the NFT DAO. So, the user would lock their NFT to vote instead of the auxiliary tokens.

Reply:
Yes, I will discuss the locking and unlocking feature with SC team in future development locking and unlocking will be done using NFT token for now to set the token allowance we need Fungible token in the NFT case which shall be removed.